app/theme/theme.js

```diff
@@ -0,0 +1,18 @@
+export const Colors =  {
+  primaryColor:           '#F5F5F5',
+  accentColor:            '#FF5722',
+  backgroundColor:        '#FAFAFA',
```

Comment:
This file will serve to create a consistent user experience. Currently it only exports `Colors`, but We could export a `Metrics` variable in the future that holds standard sizes.`Colors` can be imported into any component to maintain a consistent style throughout the app. If we use these common colors then we can easily change the theme of the whole app.